licenses/management/commands/load_html_files.py

```diff
@@ -26,19 +27,42 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser: ArgumentParser):
         parser.add_argument("input_directory")
+        parser.add_argument(
+            "--versions",
+            help="comma-separated license versions to include, e.g. '1.0,4.0'",
+        )
 
     def handle(self, input_directory, **options):
-        # We're just doing the BY 4.0 licenses for now
+        if options["versions"]:
+            versions_to_include = options["versions"].split(",")
+        else:
+            versions_to_include = None
```

Comment:
Should versions to include default to 4.0 licenses?

Reply:
If versions_to_include is not specified, we just do all versions.